### ISSUE TITLE
ValueError: Don't know how to convert data type: Int64

### ISSUE THREAD

**Issue by nrodri86** [dask/fastparquet#465]
I am trying to do a parquet file using Dask and Fastparquet from a Dataframe using a column with the type 'Int64'  (https://pandas.pydata.org/pandas-docs/stable/user_guide/integer_na.html). But unfortunately, I got the following error:

"ValueError: Don't know how to convert data type: Int64" can you help me to solve this issue.

My code basically is: 

      df_table = dd.read_sql_table(table.name,
                                     self.bd_connector.make_url_sql_server(),
                                     schema=table.schema,
                                     **table.read_params
                                     )
        log.debug("Writing the parquet file")
        table_schema = table.ingestion_schemas['table_schema']
        index_schema = table.ingestion_schemas['index_schema']
        df_table = df_table.astype(table_schema)
        df_table.astype(index_schema)
        _bytes_conveter_to_hex(df_table, table_schema)
        df_table[exclude_columns(df_table)].to_parquet(destiny,
                                                       'fastparquet',
                                                       compression='gzip',
                                                       storage_options={'token': self.gcp_cloud_storage_connector.token})

.

**Comment by martindurant** [comment]
Yep, there are a few new pandas extension data types which are not yet covered by fastparquet automatically. I am not sure, but this one might be fixable by supplying "int" as the object conversion method for the file. If you give some example code of how to make the dataframe (without a database I don't have access to), I could show you how.

**Comment by nrodri86** [comment]
Thanks Martin, I decided to use ‘float’ instead.  The thing  is that I have nulls in the column and I want  a minimalistic solution so  'float' works for me. If 'Int64' is accepted  in the fututre,  I will change the schema.  

With int I got this error.

  File "pandas\_libs\lib.pyx", line 547, in pandas._libs.lib.astype_intsafe
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

According to  the pandas documentation: 

"Because NaN is a float, this forces an array of integers with any missing values to become floating point. In some cases, this may not matter much. But if your integer column is, say, an identifier, casting to float can be problematic. Some integers cannot even be represented as floating point numbers."

**Comment by languitar** [comment]
I would also need Int64 support to represent null values in the frame. Given that my parquet file in the end goes into a postgres DB with nullable integer data type, I cannot use float, which isn't compatible for insertion into a postgres integer column.

**Comment by martindurant** [comment]
I would be happy for someone to implement handling of nullable int columns for fastparquet...

**Comment by cpitchford** [comment]
I've raised PR#483 https://github.com/dask/fastparquet/pull/483 as an attempt to add support for nullable int/bool/string dtypes in pandas 1.0.0

I'm using it with Dask to download CSV data and convert to parquet, then read back for processing and it has been behaving well. I'd like some feed back and I'm happy to improve the solution
